src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-10-studio-71.md

```diff
@@ -0,0 +1,25 @@
+---
+title: October 10, 2017
+description: This version focuses on some speed optimization. Branches listing loading time improvement, ability to remove old releases.
+tree_item_index: 999999994
+review:
+    comment: ''
+    date: '2017-10-10'
+    status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v71'}}
+This version focuses on some speed optimizations: branches listing loading time improvement, ability to remove old releases.
+{{! /multiexcerpt}}
+
+## Branches Listing Optimization
+Branches listing screen has been optimized with this version, leading to an average 90% speed gain on larger projects with a hundred branches or more. It is also made to scale: feel free to create as many more as needed, we'll handle it!
```

Comment:
`Branches listing screen` => Branch management panel

Reply:
`feel free to create as many more as needed` => feel free to create as many branches as you need